### PR TITLE
smoke tests

### PR DESCRIPTION
**Pull request by dtaylor7**
Add smoke test config and remove logs from other tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,16 +9,16 @@
     "start": "NODE_PATH=. node server.js",
     "setup": "NODE_PATH=.",
     "dev": "NODE_PATH=. NODE_ENV=development node-dev server.js",
-    "test": "NODE_PATH=. NODE_ENV=testing nyc mocha 'test/unit/**/*.test.js'",
-    "test:unit": "NODE_PATH=. NODE_ENV=testing mocha 'test/unit/**/*.js'",
+    "test": "NODE_PATH=. NODE_ENV=testing LOG_LEVEL=OFF nyc mocha 'test/unit/**/*.test.js'",
```
Comment (mattyod) [inline package.json]: Why would you want to turn off app logging when running your test suite?

File: test/e2e/codecept.smoke.conf.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-magic-numbers */
```
Comment (mattyod) [inline test/e2e/codecept.smoke.conf.js]: No magic numbers here, we can remove this.

File: package.json (hunk 1 above)
Comment (mattyod) [reply package.json]: But hopefully the logs provide useful information? So long as all the tests pass you don't have to read anything. If one fails, do you not want as much info as possible?

Comment (dtaylor7) [reply package.json]: you will get a stack and errors when the unit fails so have some information then. When debugging the error the dev can turn these on if necessary.

Comment (mattyod) [reply package.json]: on a pipeline?

Comment (dtaylor7) [reply package.json]: removed `LOG_LEVEL=OFF`  from tests

**Pull request merged**